README.md

```diff
@@ -0,0 +1,56 @@
+![Go](https://github.com/qba73/meteo/workflows/Go/badge.svg)
+[![Go Report Card](https://goreportcard.com/badge/github.com/qba73/meteo)](https://goreportcard.com/report/github.com/qba73/meteo)
+[![Maintainability](https://api.codeclimate.com/v1/badges/4afc34a390da95ed9327/maintainability)](https://codeclimate.com/github/qba73/meteo/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/4afc34a390da95ed9327/test_coverage)](https://codeclimate.com/github/qba73/meteo/test_coverage)
+
+
+# meteo
+
+Meteo is a Go client library for the weather and meteorological forecast from [Yr](https://www.yr.no/en).
```

Comment:
Great README! Maybe move the example up to the top, so it's the first thing users see?